waku/v2/protocol/filter/client.go

```diff
@@ -128,23 +131,27 @@ func (wf *WakuFilterLightNode) onRequest(ctx context.Context) func(s network.Str
 			wf.metrics.RecordError(decodeRPCFailure)
 			return
 		}
+		pubSubTopic := messagePush.PubsubTopic
 		//For now returning failure, this will get addressed with autosharding changes for filter.
 		if messagePush.PubsubTopic == nil {
-			logger.Error("empty pubsub topic")
-			wf.metrics.RecordError(decodeRPCFailure)
-			return
+			*pubSubTopic, err = getPubSubTopicFromContentTopic(messagePush.WakuMessage.ContentTopic)
```

Comment:
I'm confused, but if `messagePush.PubsubTopic` is nil, that means that `pubSubTopic` on line 134 will be nil, right? if so, wouldnt you end up with an error due to dereferencing `*pubsubTopic` which is nil?

Reply:
Yeah, took lazy approach. Fixing it by making pubSubTopic a string var.